app/controllers/job_invocations_controller.rb

```diff
@@ -0,0 +1,249 @@
+class JobInvocationsController < ApplicationController
+  def new
+    @composer = JobInvocationComposer.new(JobInvocation.new,
+                                          :host_ids => params[:host_ids],
+                                          :targeting => {
+                                            :targeting_type => Targeting::STATIC_TYPE,
```

Comment:
Shouldn't this param be taken from the params as well, otherwise we would loose the choice on validation error